src/install.coffee

```diff
@@ -40,7 +40,7 @@ class Install extends Command
              apm install --packages-file my-packages.txt
              apm i (with any of the previous argument usage)
 
-      Install the given Atom package to ~/.atom/packages/<package_name>.
+      Install the given Atom package to ~/.pulsar/packages/<package_name>.
```

Comment:
```suggestion
      Install the given Pulsar package to ~/.pulsar/packages/<package_name>.
```

We can update this from "Atom" to "Pulsar"?

Reply:
Yep, looks like this was just missed.